src/main/java/uk/gov/hmcts/reform/em/stitching/config/security/SecurityConfiguration.java

```diff
@@ -44,19 +61,36 @@ public void configure(WebSecurity web) {
 
     @Override
     protected void configure(final HttpSecurity http) throws Exception {
-        // Don't erase user credentials as this is needed for the user profile
-        final ProviderManager authenticationManager = (ProviderManager) authenticationManager();
-        authenticationManager.setEraseCredentialsAfterAuthentication(false);
-        authCheckerFilter.setAuthenticationManager(authenticationManager());
-
-        http.antMatcher("/api/**")
-            .addFilter(authCheckerFilter)
-            .sessionManagement().sessionCreationPolicy(STATELESS).and()
-            .csrf().disable()
-            .formLogin().disable()
-            .logout().disable()
-            .authorizeRequests()
-            .anyRequest()
-            .authenticated();
+
+        http.csrf()
+                .disable()
+                .addFilter(authCheckerServiceOnlyFilter)
+                .sessionManagement()
+                .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
+                .and()
+                .authorizeRequests()
+                .antMatchers("/api/**").authenticated()
+                .and()
+                .oauth2ResourceServer()
+                .jwt()
+                .and()
+                .and()
```

Comment:
this looks like a duplicate

Reply:
I tried removing the second and() call in my implementation on Anno and it causes errors

Reply:
Yes this is required